docs/modules/ROOT/partials/getting-started/proc-configuring-registry-ui.adoc

```diff
@@ -0,0 +1,36 @@
+
+[id="configuring-registry-ui"]
+= Configuring {registry} User Interface
+
```

Comment:
Suggest "{registry} web console" instead of "{registry} User Interface" or "UI" (occurs in several places) to be consistent with rest of the docs

Reply:
Done.